backend/models/create-animal.json

```diff
@@ -0,0 +1,59 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "title": "create-animal",
+    "type": "object",
+    "properties": {
+        "animalName": {
+            "type": "string"
+        },
+        "type": {
+            "type": "string",
+            "enum": ["cat", "dog", "other"]
+        },
+        "breed": {
+            "type": "string"
+        },
+        "age": {
+            "type": "string",
+            "enum": ["baby", "young", "adult", "senior"]
+        },
+        "gender": {
+            "type": "string",
+            "enum": ["n/a", "male", "female"]
+        },
+        "disposition": {
+            "type": "string",
+            "enum": ["goodWithOtherAnimals", "goodWithChildren", "mustBeLeashed"]
```

Comment:
I think disposition is supposed to be able to be any combination of these 3 so this might work better as 3 separate boolean values

Reply:
Good point. I think it would be even better if I have it as an array. The user can select from the available disposition in UI for an animal. Having it as separate boolean is not feasible if we want to add more disposition in the future. Let me know your thoughts on this.

Reply:
An array sounds good to me!

Reply:
![Screen Shot 2022-02-15 at 11 05 54 PM](https://user-images.githubusercontent.com/52226049/154214000-64fba482-9787-4cfd-9903-e42caec0b4c0.png)